src/lib/sidenav/drawer.ts

```diff
@@ -145,10 +145,22 @@ export class MdDrawer implements AfterContentInit, OnDestroy {
   private _currentTogglePromise: Promise<MdDrawerToggleResult> | null;
 
   /** Event emitted when the drawer is fully opened. */
-  @Output('open') onOpen = new EventEmitter<MdDrawerToggleResult | void>();
+  @Output() openChange = new EventEmitter<MdDrawerToggleResult | void>();
```

Comment:
I think this is really confusing now. I would expect `openChange` to fire when the drawer closes, since that is a change in the open state of the drawer. If we're going with this naming style we should get rid of the `closeChange` one and just emit booleans on the `openChange` stream whenever the drawer opens or closes.

We should also make it match the input (either both `open` or both `opened`) so that people can do 2-way binding